test/parallel/test-http-server.js

```diff
@@ -86,19 +86,19 @@ server.on('listening', function() {
   });
 
   c.on('close', function() {
-    assert.equal(c.readyState, 'closed');
+    assert.strictEqual(c.readyState, 'closed');
   });
 });
 
 process.on('exit', function() {
-  assert.equal(4, request_number);
-  assert.equal(4, requests_sent);
+  assert.strictEqual(4, request_number);
+  assert.strictEqual(4, requests_sent);
 
   var hello = new RegExp('/hello');
-  assert.equal(true, hello.exec(server_response) != null);
+  assert.notEqual(null, hello.exec(server_response));
```

Comment:
Ah yes, I missed that it was a regexp so it will always return `null` on failure. `notStrictEqual()` is better here and below.